mlir/lib/Dialect/XeGPU/Transforms/XeGPUWgToSgDistribute.cpp

```diff
@@ -29,6 +31,29 @@ using namespace mlir;
 
 namespace {
 
+static std::pair<SmallVector<int64_t>, int>
+getSgShapeAndCount(ArrayRef<int64_t> shape, xegpu::LayoutAttr layout) {
```

Comment:
can this go to XeGPUUtils so XeGPU blocking could use it?

Reply:
No, they are different logic. blocking is using inst_data, here it is using sg_layout and sg_data.